stdlib/public/core/FloatingPointRandom.swift

```diff
@@ -0,0 +1,215 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Since you created new files would it have to be added in [CMakeLists.txt](https://github.com/apple/swift/blob/21b91cd03c1ac04d093945a02aacceaba800eee5/stdlib/public/core/CMakeLists.txt#L72) as well? Also in [GroupInfo.json](https://github.com/apple/swift/blob/master/stdlib/public/core/GroupInfo.json)?

Reply:
I don’t know how those files work, but I’m taking your word for it so I’ve added FloatingPointRandom.swift to both of them.

Reply:
I'm a bit mystified how it compiled and passed tests without doing this...

Reply:
Yeah, as far as I know at least the CMakeLists.txt is required to build, but GroupInfo I just a guess that is for grouping the files when generating Xcode target or it does something else @airspeedswift?

Reply:
My recollection is that the build will fail if anything isn't also in GroupInfo (because there's an explicit check that everything appears there). This may have changed in the last few years, however.